<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-1490L1R" failures="1" tests="5" name="Default test" time="43.89" errors="0" timestamp="2025-02-03T20:30:20 GMT-05:00">
  <testcase classname="testcases.TrialOne" name="login" time="10.906"/>
  <testcase classname="testcases.TrialOne" name="navigateToCCReport" time="7.332"/>
  <testcase classname="testcases.TrialOne" name="switchToTab" time="5.242">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;functions.ReadingDataFromExcel.readExcel()&amp;quot; because &amp;quot;this.readFromExcel&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "functions.ReadingDataFromExcel.readExcel()" because "this.readFromExcel" is null
at testcases.TrialOne.switchToTab(TrialOne.java:68)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- switchToTab -->
  <testcase classname="testcases.TrialOne" name="verifyReportBody" time="0.083"/>
  <testcase classname="testcases.TrialOne" name="navigateToBoosterReport" time="10.164"/>
</testsuite> <!-- Default test -->
